provisioner.go

```
@@ -193,11 +195,17 @@ func (p *LocalPathProvisioner) Provision(opts pvController.ProvisionOptions) (*v
 	path := filepath.Join(basePath, folderName)
 	logrus.Infof("Creating volume %v at %v:%v", name, node.Name, path)
 
+	storage := pvc.Spec.Resources.Requests[v1.ResourceName(v1.ResourceStorage)]
+	iSizeInBytes, _ := storage.AsInt64()
```

Better not ignore the error.